rmgpy/rmg/input.py

```diff
@@ -1153,7 +1153,8 @@ def model(toleranceMoveToCore=None, toleranceMoveEdgeReactionToCore=np.inf, tole
           ignoreOverallFluxCriterion=False,
           maxNumSpecies=None, maxNumObjsPerIter=1, terminateAtMaxObjects=False,
           toleranceThermoKeepSpeciesInEdge=np.inf, dynamicsTimeScale=(0.0, 'sec'),
-          toleranceBranchReactionToCore=0.0, branchingIndex=0.5, branchingRatioMax=1.0):
+          toleranceBranchReactionToCore=0.0, branchingIndex=0.5, branchingRatioMax=1.0,
+          toleranceTransitoryDict={}):
```

Comment:
What is this for again? I'm confused, why do we need this?

Reply:
toleranceTransitoryDict has the transitory sensitivity tolerance for each species of interest.